app/src/main/java/org/gdg/frisbee/android/fragment/FirstStartStep1Fragment.java

```diff
@@ -50,6 +51,7 @@
 
 public class FirstStartStep1Fragment extends Fragment {
 
+    private static final String ARG_SELECTED_CHAPTER = "selected_chapter";

```

Comment:
Why is this redefined here? Can this definition in MainActivity be made public and reused here?